tests/test_benchmarks.py

```diff
@@ -330,6 +330,57 @@ def t():
         v.validate_json(json_data[1])
 
 
+frozenset_of_ints_data = ({i for i in range(1000)}, {str(i) for i in range(1000)})
+
+
+@skip_pydantic
+@pytest.mark.benchmark(group='frozenset of ints')
+def test_frozenset_of_ints_pyd(benchmark):
+    class PydanticModel(BaseModel):
+        __root__: FrozenSet[int]
+
+    @benchmark
+    def t():
+        PydanticModel.parse_obj(set_of_ints_data[0])
+        PydanticModel.parse_obj(set_of_ints_data[1])
```

Comment:
```suggestion
    benchmark(PydanticModel.parse_ob, set_of_ints)
```
not sure we need both ints and strings, just parsing ints should tell us about performance.